src/main/groovy/net/neoforged/gradleutils/GradleUtils.groovy

```diff
@@ -240,6 +241,7 @@ class GradleUtils {
         }
     }
 
+    @UntrackedTask(because = 'We always want to configure CI')
```

Comment:
Should probably make it more neutral, like `CI configuration should always run in every build`.